.g8/datePage/app/forms/$className$FormProvider.scala

```diff
@@ -8,13 +8,5 @@ import play.api.data.Form
 
 class $className$FormProvider @Inject() extends Mappings {
 
-  def apply(): Form[LocalDate] =
-    Form(
-      "value" -> localDate(
-        invalidKey     = "$className;format="decap"$.error.invalid",
-        allRequiredKey = "$className;format="decap"$.error.required.all",
-        twoRequiredKey = "$className;format="decap"$.error.required.two",
-        requiredKey    = "$className;format="decap"$.error.required"
-      )
-    )
+  def apply(): Form[LocalDate] = Form("value" -> localDate(invalidKey = "$className;format="decap"$.error.invalid"))
```

Comment:
Wasn't sure whether I'm actually supposed to tinker with these .g8 temples, but I obviously have done. Correct me if I need to change it back.